addons/website/static/src/scss/website.scss

```diff
@@ -1433,6 +1433,21 @@ $ribbon-padding: 100px;
     :not(.o_cookies_popup) {
         bottom: 0;
     }
+    .modal{
+        overflow: auto;
+        &.o_cookies_discrete{
+            .modal-dialog{
+                margin: 0;
+                .o_cc{
+                    .text-right{
+                        @include media-breakpoint-down(md){
+                            bottom: 16px;
+                        }
+                    }
+                }
+            }
+        }
+    }
```

Comment:
I doubt this is the right fix for multiple reasons:

- Normal popup snippet can scroll just fine, like any normal modal. Cookie popup being popup snippets, something is breaking the scroll specifically... I doubt **adding** scss code will be the solution

- The SCSS is ultra specific for a generic problem: o_cookies_discrete (while I tested non discrete and it indeed cannot scroll), o_cc which is a color system class and should thus never be related to a px alignment, text-right is a horizontal alignment utility class it should definitely never be targeted by a CSS rule changing vertical alignment.

Reply:
Thanks for the feedback, it pointed me in the right direction. Now I think the solution is the correct one.